File: devops-scm-api/src/main/java/com/tencent/devops/scm/api/pojo/IssueListOptions.java
package com.tencent.devops.scm.api.pojo;

import com.tencent.devops.scm.api.enums.IssueState;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class IssueListOptions {
    private Integer page;
    private Integer pageSize;
    private IssueState state;
}

File: devops-scm-api/src/main/kotlin/com/tencent/devops/scm/api/pojo/IssueListOptions.kt
package com.tencent.devops.scm.api.pojo

import com.tencent.devops.scm.api.enums.IssueState

data class IssueListOptions(
    val page: Int? = null,
    val pageSize: Int? = null,
    val state: IssueState? = null
)
